examples/bbm_bbm_1d/bbm_bbm_1d_basic_reflecting.jl

```diff
@@ -0,0 +1,48 @@
+using OrdinaryDiffEq
+using DispersiveShallowWater
+using SummationByPartsOperators: MattssonNordström2004, derivative_operator
+
+###############################################################################
+# Semidiscretization of the BBM-BBM equations
+
+equations = BBMBBMEquations1D(gravity_constant = 9.81, D = 1.0)
+
+initial_condition = initial_condition_manufactured_reflecting
+source_terms = source_terms_manufactured_reflecting
+boundary_conditions = boundary_condition_reflecting
+
+# create homogeneous mesh
+coordinates_min = 0.0
+coordinates_max = 1.0
+N = 512
+mesh = Mesh1D(coordinates_min, coordinates_max, N)
+
+# create solver with SBP operators of accuracy order 4
+accuracy_order = 4
+D1 = derivative_operator(MattssonNordström2004(),
+                         derivative_order = 1, accuracy_order = accuracy_order,
+                         xmin = mesh.xmin, xmax = mesh.xmax, N = mesh.N)
+D2 = derivative_operator(MattssonNordström2004(),
+                         derivative_order = 2, accuracy_order = accuracy_order,
+                         xmin = mesh.xmin, xmax = mesh.xmax, N = mesh.N)
+solver = Solver(D1, D2)
```

Comment:
`D2` is not used here, is it? Can we clarify this by passing `nothing` or something like that instead - or add a comment?

Reply:
`D2` is used [for the Dirichlet boundary condition](https://github.com/JoshuaLampert/DispersiveShallowWater.jl/pull/104/files#diff-014e2648af7ba655effcd1aaea79591f8e82335301f5be67fecfc6c80ecb834bR150).

Reply:
Right, I just thought about the Neumann one. Sorry